samples/snippets/src/main/java/com/example/bigtable/deletes/DeleteColumnFamilyExample.java

```diff
@@ -17,16 +17,18 @@
 package com.example.bigtable.deletes;
 
 // [START bigtable_delete_column_family]
-import com.google.cloud.bigtable.data.v2.BigtableDataClient;
-import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.models.ModifyColumnFamiliesRequest;
 import java.io.IOException;
 
 public class DeleteColumnFamilyExample {
-  public void deleteColumnFamily(String projectId, String instanceId, String tableId)
-      throws IOException {
-    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
-      dataClient.mutateRow(
-          RowMutation.create(tableId, "phone#5c10102#20190501").deleteFamily("stats_summary"));
+  public void deleteColumnFamily(
+      String projectId, String instanceId, String tableId, String columnFamily) throws IOException {
+    try (BigtableTableAdminClient tableAdminClient =
+        BigtableTableAdminClient.create(projectId, instanceId)) {
+      ModifyColumnFamiliesRequest modifyColumnFamiliesRequest =
+          ModifyColumnFamiliesRequest.of(tableId).dropFamily(columnFamily);
```

Comment:
I thought we decided to inline the string here to not just be all variables. I'm fine either way, just want to point it out.